dnf/base.py

```diff
@@ -61,7 +61,9 @@
 import dnf.transaction
 import dnf.util
 import dnf.yum.rpmtrans
+import errno
 import functools
+import gi
```

Comment:
please use `from gi.repository import Modulemd`

Reply:
Will do. It needs to be used together with gi.require_version() I suppose to prevent module loading order issues.